index.js

```diff
@@ -10,6 +10,7 @@ var UnrarStream = require('./lib/stream.js');
 var Unrar = function(options) {
   this._arguments = options.arguments || [];
   this._filepath = options.path || options;
+  this._unrarBin = options.$bin || false;
```

Comment:
Please do not use same property for two meanings (using custom binary path and binary path itself)

Reply:
Sorry, i mean using same property as string and as boolean. Using empty string for for absent of custom path is more preferable.

Reply:
I will also apply this recommendation.